src/Rotations.jl

```diff
@@ -1,5 +1,47 @@
+# we generate code in this module, so precompile where possible
+ VERSION >= v"0.4.0-dev+6521" && __precompile__(true)
```

Comment:
The `VERSION` check is probably not required here - we don't support 0.3, right?